artview/components/plot_radar.py

```diff
@@ -44,6 +44,7 @@ class RadarDisplay(Component):
     Vgatefilter = None  #: see :ref:`shared_variable`
     VplotAxes = None  #: see :ref:`shared_variable` (no internal use)
     VpathInteriorFunc = None  #: see :ref:`shared_variable` (no internal use)
+    VpyartDisplay = None  #: see :ref:`shared_variable`
```

Comment:
I wholeheartedly agree with everything you said. Let's keep an eye out for better ways in the future with working functionality now. Some performance I think we'll have to live with as a function of reasonably large data files (i.e. number of points).